subprojects/test-kit/src/integTest/groovy/org/gradle/testkit/runner/BaseGradleRunnerIntegrationTest.groovy

```diff
@@ -177,6 +177,12 @@ abstract class BaseGradleRunnerIntegrationTest extends AbstractIntegrationSpec {
     }
 
     static String determineMinimumVersionThatRunsOnCurrentJavaVersion(String desiredGradleVersion) {
+        if (JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_16)) {
+            def compatibleVersion = GradleVersion.version("7.0-20210224105427+0000")
```

Comment:
Can't you use "7.0" here already? If yes, it would make sense to change this to "7.0" directly now.

Reply:
Tried it, does not work, unfortunately.

Reply:
I guess the `return compatibleVersion.version` is an issue. Lets keep it like this for now.